scss/navbar.scss

```diff
@@ -0,0 +1,20 @@
+.train-header {
+  background-color: $training-white;
+  box-shadow: 0 1px 15px $training-grey;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  padding: 30px;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+
+  .logo{
+    height: 30px;
+  }
+
+  .dev{
+    align-self: center;
```

Comment:
I'd use align items in the parent element to center all children instead

Reply:
but the logo is not aligned center, only this element